tests/integration/bigquery.spec.ts

```diff
@@ -137,6 +137,47 @@ suite("@dataform/integration/bigquery", ({ after }) => {
     expect(exampleAssertionFail).to.be.eql(undefined);
 
     expect(persistedMetaData.length).to.be.eql(9);
+
+    // metadata
+    const target = {
```

Comment:
we should be able to get this out of the `ExecutionGraph` instead of re-creating it